app/views/articles/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>New article</h1>
+
+<div class="flex flex-wrap -mx-3 mb-6">
+  <%= form_with(model: @article) do |f| %>
+    <% if @article.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(@article.errors.count, "error") %> prohibited this article from being saved:</h2>
```

Comment:
Can we move this string into a locale file? It's a good practice to prepare ourselves for internationalisation (i18n) from the beginning. It also separates our content from our behaviour.

(ditto throughout)